PowerEditor/installer/nativeLang/korean.xml

```diff
@@ -429,61 +429,61 @@
   <Item id="1904" name="저장"/> 
 </Run>
 
-<MD5FromFilesDlg title="파일의 MD5 요약 생성">
-  <Item id="1922" name="MD5를 생성할 파일 선택..."/>
-  <Item id="1924" name="클립보드로 복사"/>
+<MD5FromFilesDlg title="기록철의 MD5 요약 생성">
+  <Item id="1922" name="MD5를 생성할 기록철 선택..."/>
+  <Item id="1924" name="오림판으로 복사"/>
   <Item id="2"    name="닫기"/>
 </MD5FromFilesDlg>
 
 <MD5FromTextDlg title="MD5 요약 생성">
   <Item id="1932" name="각 줄을 분리된 문자열로 처리"/>
-  <Item id="1934" name="클립보드로 복사"/>
+  <Item id="1934" name="오림판으로 복사"/>
   <Item id="2"    name="닫기"/>
 </MD5FromTextDlg>
 
-<SHA256FromFilesDlg title="파일의 SHA-256 요약 생성">
-  <Item id="1922" name="SHA-256을 생성할 파일 선택..."/>
-  <Item id="1924" name="클립보드로 복사"/>
+<SHA256FromFilesDlg title="기록철의 SHA-256 요약 생성">
+  <Item id="1922" name="SHA-256을 생성할 기록철 선택..."/>
+  <Item id="1924" name="오림판으로 복사"/>
   <Item id="2"    name="닫기"/>
 </SHA256FromFilesDlg>
 
-<SHA256FromTextDlg title="파일의 SHA-256 요약 생성">
+<SHA256FromTextDlg title="기록철의 SHA-256 요약 생성">
   <Item id="1932" name="각 줄을 분리된 문자열로 처리"/>
-  <Item id="1934" name="클립보드로 복사"/>
+  <Item id="1934" name="오림판으로 복사"/>
   <Item id="2"    name="닫기"/>
 </SHA256FromTextDlg>
 
-<PluginsAdminDlg title="플러그인 관리" titleAvailable = "사용 가능" titleUpdates = "업데이트" titleInstalled = "설치됨">
+<PluginsAdminDlg title="플러그인 관리" titleAvailable = "사용 가능" titleUpdates = "갱신" titleInstalled = "설치됨">
     <ColumnPlugin   name="플러그인"/>
```

Comment:
Isn't there any translation for plugin in Korean? If not, I really hope you to close this PR.